.github/ISSUE_TEMPLATE/2_new-hub.yml

```
@@ -115,56 +67,55 @@ body:
       options:
         - GitHub Authentication (e.g., @mygithubhandle)
         - Google OAuth (e.g., myemailaddress@2i2c.org)
+        - CILogon (yy
         - Username / Password via [auth0](https://auth0.com/docs/connections/database)
         - Other (may not be possible, please specify in comments)
     validations:
       required: true
 
-  - type: input
+  - type: textarea
     id: hub-logo
     attributes:
-      label: Hub logo
+      label: Hub logo information
       description: |
-        A URL to an image that will be used on the hub's landing page. (optional)
-    validations:
-      required: false
-
-  - type: input
-    id: hub-logo-url
-    attributes:
-      label: Hub logo URL
-      description: |
-        A URL that the hub's logo will point to when users click on it on the landing page (optional)
+        The hub landing page will have its own logo and a link attached to it. This generally points to a community or institutional website.
```

Maybe rephrase this to make it more clear what we mean?
```suggestion
        The hub landing page can have a specific logo and a link attached to it. This generally points to a community or institutional website.
```